src/tdl/queue/processing_rules.py

```diff
@@ -24,7 +24,7 @@ def get_response_for(self, request):
             result = processing_rule.user_implementation(*request.params)
             return ValidResponse(request.id, result)
         except Exception as e:
-            print(e.message)
+            print(e)
```

Comment:
If this going to print the entire stacktrace? If that is the case, then this is not desirable. We only want to provide a hint not the full stack.